packages/cells/src/cells/date-picker-cell.tsx

```diff
@@ -1,15 +1,80 @@
-import * as React from "react";
-import { CustomCell, CustomRenderer, drawTextCell, GridCellKind } from "@glideapps/glide-data-grid";
+import React from "react";
+import { assertNever } from "../../../core/src/common/support";
```

Comment:
Is this the way to do this?

Reply:
fuck...

Reply:
duplicate that method into the cells package.

Reply:
just the method or the whole support.ts? For now, I will do just that method :)

Reply:
👍